test/verify/check-system-realms

```diff
@@ -83,7 +83,12 @@ class CommonTests:
         # Tell realmd to enable domain-qualified logins; unqualified ones are covered in testUnqualifiedUsers
         m.write("/etc/realmd.conf", "[cockpit.lan]\nfully-qualified-names = yes\n", append=True)
 
-        self.login_and_go("/system")
+        # Test that we reconnect on privileges change
```

Comment:
This is a bit awkward test case - I used this as reproducer. @martinpitt WDYT?

Reply:
Seems fine to me -- certainly better than joining yet another time, as that's quite expensive.